scripts/transpile.js

```diff
@@ -66,43 +70,51 @@ if (spliceLength == 0) {
 }
 
 console.log(`Spawning ${cpuCount} processes to transpile ${fileList.length} files.`);
-for (let i = 1; i <= cpuCount; i+=1) {
+for (let i = 1; i <= cpuCount; i += 1) {
   if (fileList.length < spliceLength || i === cpuCount) {
     spliceLength = fileList.length;
   }
   let filesSubset = fileList.splice(0, spliceLength).join(',');
-  const transpiler = childProcess.spawn(
-    'node', ['scripts/transpile.js', '--spawned', '--files', filesSubset]);
+  const transpiler = childProcess.spawn('node', [
+    'scripts/transpile.js',
+    '--spawned',
+    '--files',
+    filesSubset
+  ]);
   const onData = data => console.log(`${data}`);
-  transpiler.stdout.on('data',onData);
+  transpiler.stdout.on('data', onData);
   transpiler.stderr.on('data', onData);
   transpiler.on('close', code => {
     console.log(`child process exited with code ${code}`);
     touchCanary();
   });
 }
 
-function transpile (fileList) {
+function transpile(fileList) {
   fileList.forEach(file => {
     const fileParts = file.split('/');
     const fileName = fileParts.pop();
     const directory = rootDir + fileParts.join('/').replace('/src/', '/build/');
     const fullPath = `${directory}/${fileName}`;
 
-    fs.readFile(rootDir + file, {
-      encoding: 'utf-8'
-    }, (err, data) => {
-      console.log('Transpiling', fullPath);
-      mkdirp.sync(directory);
-      const full = babel.transform(data, { plugins, compact: false });
-      fs.writeFile(fullPath, full.code, writeHandler);
-      const min = babel.transform(data, {
-        presets: ['babel-preset-babili'],
-        plugins,
-        compact: true,
-        comments: false
-      });
-      fs.writeFile(`${fullPath.replace('.js', '-min.js')}`, min.code, writeHandler);
-    });
+    fs.readFile(
```

Comment:
no no no, stay out of my nodejs prettier!

Reply:
@barrymcgee @hatched I assume we can tell prettier to stay out of certain paths? I'm not sure this needs to be treated differently from the rest of the code, but if it's a problem perhaps that's something we can do?

Reply:
@jaycee Yes, there is a `.prettierignore` file in root which will take any file or folder glob/path and respect it.